manifests/mysqld_exporter.pp

```diff
@@ -137,7 +137,7 @@
     notify  => $notify_service,
   }
 
-  $options = "-config.my-cnf=${cnf_config_path} ${extra_options}"
+  $options = "--config.my-cnf==${cnf_config_path} ${extra_options}"
```

Comment:
this change would make it backwards incompatible. Can you implement a selector statement that uses the old or the new syntax, depending on the version?